clerk-tools.lic

```diff
@@ -0,0 +1,91 @@
+
+custom_require.call(%w[drinfomon equipmanager common common-crafting common-items common-travel])
+
+class Clerk
```

Comment:
💡 I recommend making the class name match the file name when possible (e.g. `class ClerkTools`)